truffle/src/main/ruby/core/shims.rb

```diff
@@ -216,5 +193,16 @@ def self.synchronize(object)
 
 end
 
-$PROGRAM_NAME = $0
+module Errno
 
+  # TODO CS 18-Apr-15 this should be a separate class
+  DomainError = EDOM
+
+end
+
+module Math
+  DomainError = Errno::EDOM
+end
+
+$PROGRAM_NAME = $0
+$$ = Process.pid
```

Comment:
I guess the real question you're asking is why am I setting this statically when it could change? Well that's why it's in `shims.rb` where there is no shame.